internal/configs/virtualserver.go

```diff
@@ -499,9 +499,21 @@ func generateBool(s *bool, defaultS bool) bool {
 	return defaultS
 }
 
+func generatePath(path string) string {
+	// Wrap the regular expression (if present) inside double quotes (") to avoid NGINX parsing errors
+	if strings.HasPrefix(path, "~*") {
+		return fmt.Sprintf(`~* "%v"`, strings.TrimPrefix(strings.TrimPrefix(path, "~*"), " "))
```

Comment:
is it possible to trim prefix only once, for `"~* "` ?
same for the `~`

or that whitespace is optional?

Reply:
The whitespace is optional, is just there in case the user adds it, otherwise, this: `~* regex` will create: `~* " regex"` note the space before `regex` should not be part of the regex inside `"`